internal/js_library/test/strip_prefix/test.js

```diff
@@ -1,9 +1,9 @@
 // Try to load on unstripped path, this should fail
 try {
-    require("stripped_library/library/file");
+  require('stripped_library/library/file');
 } catch (exc) {
-    if (exc.code !== 'MODULE_NOT_FOUND') throw exc;
+  if (exc.code !== 'MODULE_NOT_FOUND') throw exc;
 }
 
 // Load with the stripped path, this should succeed
-require("stripped_library/file");
+require('stripped_library/file');
```

Comment:
Change came from a "yarn format" which was requested by a pre-commit hook when I merged in the latest stable branch. Can revert if needed